src/Component/Cart.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+function Cart ({cart, subtotal}) {
+    return (
+        <div className="">
+        <h2>Enrolled</h2>
+        <ul>
+            {cart.map((item)=>{
+                return <li>{item.name} ${item.price}</li>
```

Comment:
The tests are picky. They specify that there should be a colon between item.name and item.price in each li.